XRTK-Core/Packages/com.xrtk.core/Definitions/Devices/SupportedControllerType.cs

```diff
@@ -22,6 +22,8 @@ public enum SupportedControllerType
         Mouse,
         Lumin,
         OculusGo,
-        OculusQuest
+        OculusQuest,
+        ArticulatedHand,
+        GGVHand
```

Comment:
Is there a better name for this rather than this acronym ?

Reply:
What's the difference between GGVHand and ArticulatedHand?